src/main/java/seedu/address/model/EntityList/MentorList.java

```diff
@@ -0,0 +1,108 @@
+package seedu.address.model.EntityList;
+
+import java.util.ArrayList;
+import java.util.List;
+import seedu.address.model.Entity.Entity;
+import seedu.address.model.Entity.ID;
+import seedu.address.model.Entity.Mentor;
+import seedu.address.model.Entity.PrefixType;
+
+public class MentorList extends EntityList {
+    private List<Mentor> mentors;
+
+    /**
+     * Constructor.
+     *
+     */
+    public MentorList() {
+        super();
+        this.mentors = new ArrayList<>();
+    }
+
+    /**
+     * Gets Mentor by ID.
+     *
+     * @param id
+     * @return
+     */
+    @Override
+    public Mentor get(ID id) {
+        // TODO
+        return null;
+    }
+
+    /**
+     * Updates Mentor by ID.
+     *
+     * @param mentor
+     * @throws Exception if error while updating
+     */
+    @Override
+    public void update(Entity mentor) throws Exception {
```

Comment:
Wanna take a look at my ListIssueCommand#execute()? I think update() should take two params, Id idToEdit and Entity editedMentor

Reply:
Can, it's also in line with the current code.